ApolloExtra.lua

```diff
@@ -3260,103 +3351,53 @@ OlympusLootData.ApolloUpgrade = {
-			},
-			ApolloMakeUp07 =
-			{
-                Name = "ApolloMakeUp07",
-				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0137",
-					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "I'd say this little squabble can be put to bed, for now. I'm better for the moment, but I was very angry with you, love!" },
-			},
-			ApolloMakeUp08 =
-			{
-                Name = "ApolloMakeUp08",
-				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0138",
-					Emote = "PortraitEmoteCheerful",
-					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "I cannot bring myself to stay upset with you forever, little godling, so let's move along from that small misunderstanding there." },
-			},
-			ApolloMakeUp09 =
-			{
-                Name = "ApolloMakeUp09",
-				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0139",
-					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "There is no common sense with love, godling, so... why don't we pretend your indiscretion there did not occur." },
-			},
-			ApolloMakeUp10 =
-			{
-                Name = "ApolloMakeUp10",
-				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0140",
-					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "It is through times of difficulty that the love we share is tested and reforged. But you just watch yourself when testing it like that." },
-			},
-			ApolloMakeUp11 =
-			{
-                Name = "ApolloMakeUp11",
-				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0141",
-					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Love and jealousy go hand in hand, don't they, dearest? Although the former's much more pleasant, isn't it." },
-			},
-			ApolloMakeUp12 =
-			{
-                Name = "ApolloMakeUp12",
-				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				{ Cue = "/VO/Apollo_0217",
-					Emote = "PortraitEmoteAffection",
+				{ Cue = "/VO/Apollo_0115",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "Oh, all this fighting! It's just not like us at {#DialogueItalicFormat}all{#PreviousFormat}, dearest! Let's say we kissed and then made up, all right?" },
+					Text = "You know respect comes both way, right, Zagzag... I am willing to do my part, if you do yours." },
```

Comment:
"You know respect goes both ways, right, Zagzag? I am willing to do my part, if you do yours."